### PR TITLE
Update newline

### PR DESCRIPTION
**Pull request by Prithivi-Raj**
hey you

### CODE REVIEW

**Comment by Prithivi-Raj** [comment]
jjijijijij

**Comment by Prithivi-Raj** [comment]
jnjnknnkn

**Comment by Prithivi-Raj** [comment]
ed2d2ed2ed

File: newline [hunk 1]
```diff
@@ -1,2 +1,3 @@
 helloo 
 berries
+mmllmm
```
Comment (niharikachavati) [inline newline:3]: this is nice

**Pull request merged**